src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -122,16 +125,51 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
    * This builds the form with data from the previous pages
    * EUI-3732 - Breathing space data not persisted on Previous button click with ExpUI Demo
    */
-  toPreviousPage() {
+  public toPreviousPage(): void {
+    this.validationErrors = [];
     let caseEventData: CaseEventData = this.buildCaseEventData();
     caseEventData.data = caseEventData.event_data;
     this.updateFormData(caseEventData);
     this.previous();
     CaseEditPageComponent.setFocusToTop();
   }
 
+  // Adding validation message to show it as Error Summary
+  public generateErrorMessage(fields: CaseField[]): void {
+    fields.filter(casefield => !this.caseFieldService.isReadOnly(casefield))
+          .filter(casefield => !this.pageValidationService.isHidden(casefield, this.editForm))
+          .forEach(casefield => {
+            const fieldElement = this.editForm.controls['data'].get(casefield.id);
+            if (fieldElement) {
+              if (fieldElement.hasError('required')) {
+                this.validationErrors.push({id: casefield.id, message: `${casefield.label} is required`});
+                fieldElement.markAsTouched();
+              } else if (fieldElement.hasError('pattern')) {
+                this.validationErrors.push({id: casefield.id, message: `${casefield.label} is not valid`});
+                fieldElement.markAsTouched();
+              } else if (fieldElement.hasError('minlength')) {
+                this.validationErrors.push({id: casefield.id, message: `${casefield.label} required minimum length`});
```

Comment:
This phrasing differs from the other three in that is the past tense whilst the others are present tense. It's also awkwardly phrased, along with the "exceeds maximum length". Shouldn't that be "exceeds the maximum length"?

As for this one, "is below the minimum length"?

Reply:
I was using the existing error message; now I have changed as mentioned above